ember/app/components/version-detailed.gjs

```diff
@@ -0,0 +1,108 @@
+import { LinkTo } from '@ember/routing';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import formatDate from 'ember-intl/helpers/format-date';
+import UkIcon from 'ember-uikit/components/uk-icon';
+import _ from 'lodash';
+
+import {
+  ProjectCell,
+  MaintainersCell,
+  EndOfLifeCell,
+  DependencyCell,
+} from './cells';
+import EOLForm from './eol-form';
+import Table from './table';
+
+import { statusToClass } from 'outdated/utils';
+
+export default class VersionDetailedComponent extends Component {
+  @service store;
+
+  get projects() {
+    return this.store
+      .peekAll('project')
+      .filter((project) =>
+        project.versionedDependencies.find(
+          (version) => version.id === this.args.version.id,
+        ),
+      );
+  }
+
+  get data() {
+    return this.projects.map((project) => ({
+      component: <template>
+        <tr class={{statusToClass project.status}}>{{yield}}</tr>
+      </template>,
+      values: {
+        name: <template><ProjectCell @project={{project}} /></template>,
+        maintainers: project.maintainers.length
+          ? <template>
+              <MaintainersCell @maintainers={{project.maintainers}} />
+            </template>
+          : undefined,
+      },
+    }));
+  }
+
+  get cells() {
+    const version = this.args.version;
+    const dependency = version.releaseVersion.dependency;
+    return Object.entries({
+      dependency: <template>
+        <DependencyCell @dependency={{dependency}} />
+      </template>,
+      releaseVersion: <template>{{formatDate version.releaseDate}}</template>,
+      endOfLife: <template>
+        <EndOfLifeCell
+          class={{statusToClass version.status}}
+          @version={{version}}
+        />
+      </template>,
+    }).map(([label, value]) => ({ label: _.startCase(label), value }));
+  }
+
+  <template>
+    <div class='detailed-header'>
+      <h1>
+        <LinkTo
+          @route='dependencies.detailed'
+          @model={{@version.releaseVersion.dependency.id}}
+          data-test-version-name
+        >
+          {{@version.name}}
+        </LinkTo>
+        {{@version.version}}
+      </h1>
+
+      <a href='{{@version.url}}' data-test-version-link>
+        <UkIcon @icon='link' @ratio={{3}} />
+      </a>
+    </div>
+
+    <hr class='seperator' />
+    <div class='uk-margin-top'>
+      <ul
+        class='uk-list uk-flex uk-flex-between uk-flex-stretch uk-margin-remove'
+      >
+        {{#each this.cells as |cell|}}
+          <li class='uk-margin-remove'><strong
+              class='uk-display-block'
+            >{{cell.label}}</strong><span
```

Comment:
Could we *decamelize* those labels? `releaseVersion` doesn't look fancy.

Reply:
those are decamelized by this `startCase`
https://github.com/adfinis/Outdated/blob/7962d572dcb4209a9ebc65de6cfbb6ada7e708d7/ember/app/components/version-detailed.gjs#L62

![image](https://github.com/adfinis/Outdated/assets/110528300/2833690f-e39a-4671-8ef8-7cdf85757553)

Reply:
sweet :ok_hand: